index.html

```diff
@@ -1189,7 +1189,7 @@ <h4>EcdsaSecp256k1VerificationKey2019</h4>
                 <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Ecdsa Secp256k1 Signature 2019</a>
```

Comment:
You could change this as well... 

... https://w3id.org/security/suites/secp256k1-2019

confusing to see https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/ point to a different place.... that would need to be fixed in perma-id repo.

Reply:
Not sure I understand what you are proposing here. Almost all of the other "Normative Definition" column links under "6.1 Verification method types" are `https://w3c-ccg.github.io/...` URLs, not ` https://w3id.org/...` URLs.

If those "should" also be w3id.org links instead, i'd do that all in a separate PR to keep this PR small and simple.

Reply:
Yes, agreed... I am pointing out that w3id.org is redirecting to those github pages repos... and that its therefore safer (from a long term link perspective, to assume w3id.org will outlast github, other wise we would not be using it).

In either case, you can't fix this here... you would need to open PRs on w3id.org first, and then come here to update the links that are currently to github.

We did get objections related to linking directly to github in previous reviews from W3C.

Reply:
All the `w3c-ccg.github.io` links should be replaced by `w3id.org` links that redirect to the `w3c-ccg.github.io` location today, and are expected to be redirected to `w3c.org/TR` at some point.  The point of the `w3id.org` links is that they will not need to change between Drafts and Recs, so anyone who implements based on these links will not have to change anything when we relocate (for instance, when we transition from Draft to Rec) the specs. On the other hand, anyone who implements based on the `w3c-ccg.github.io` links will have to change things anytime we transition from Draft to Rec or otherwise relocate the docs.

Reply:
I created a github issue capturing this task and conversation: https://github.com/w3c/did-spec-registries/issues/526

Please put further conversation over there. I'm going to "resolve" this thread so that this PR can move ahead.